src/backend/backconfig.c

```diff
@@ -110,7 +112,6 @@ void out_config_init(
         config.flags |= CFGalwaysframe; // PIC needs a frame for TLS fixups
     }
     config.objfmt = OBJ_ELF;
-    config.ehmethod = dwarfeh ? EH_DWARF : EH_DM;
```

Comment:
I hope this means we can easily disable the feature if it doesn't survive the beta testing.